examples/basic.html

```diff
@@ -132,6 +132,7 @@ <h3>450 €</h3>
           hideIfNotEligible: false,
           transitionDelay: 5500,
           suggestedPaymentPlan: [10, 4],
+          condensedView: false,
```

Comment:
I feel like this naming is not clear.
Maybe something like `oneLiner` ? wdyt ?

Reply:
This was the previous name i used 😵 we discussed it with @gautierlabarre